Networking/NetworkingTests/Responses/coupon-minimal.json

```diff
@@ -0,0 +1,43 @@
+{
+    "data": {
+        "id": 10714,
+        "code": "test",
+        "amount": "0.00",
+        "date_created": "2021-04-13T08:26:25",
+        "date_created_gmt": "2021-04-13T08:26:25",
+        "date_modified": "2021-04-13T08:26:25",
+        "date_modified_gmt": "2021-04-13T08:26:25",
+        "discount_type": "fixed_cart",
+        "description": "",
+        "date_expires": null,
+        "date_expires_gmt": null,
+        "usage_count": 0,
+        "individual_use": false,
+        "product_ids": [],
+        "excluded_product_ids": [],
+        "usage_limit": null,
+        "usage_limit_per_user": null,
+        "limit_usage_to_x_items": null,
+        "free_shipping": false,
+        "product_categories": [],
+        "excluded_product_categories": [],
+        "exclude_sale_items": false,
+        "minimum_amount": "0.00",
+        "maximum_amount": "0.00",
+        "email_restrictions": [],
+        "used_by": [],
+        "meta_data": [],
+        "_links": {
+            "self": [
+                {
+                    "href": "https://example.com/wp-json/wc/v3/coupons/720"
+                }
+            ],
+            "collection": [
+                {
+                    "href": "https://example.com/wp-json/wc/v3/coupons"
+                }
+            ]
+        }
+    }
+}
```

Comment:
This is probably not needed since the test is included it in `CouponListMapper`. Up to you if you want to maintain it. ;)